torchci/test/drciBot.test.ts

```diff
@@ -30,7 +30,10 @@ describe("verify-drci-functionality", () => {
       })
       .reply(200)
       .post("/repos/zhouzhuojie/gha-ci-playground/issues/31/comments", (body) => {
-        expect(body).toMatchObject({ body: "<!-- drci-comment-start -->hello there!<!-- drci-comment-end -->" });
+        const comment = body.body;
+        expect(comment.includes("<!-- drci-comment-start -->")).toBeTruthy();
```

Comment:
you can maybe export the const drciCommentStart and then import it here to do the comparison. Might be a bit easier to maintain.